pkg/commands/config.go

```diff
@@ -98,6 +99,15 @@ func getBaseImage(bo *options.BuildOptions) build.GetBase {
 		if bo.InsecureRegistry {
 			nameOpts = append(nameOpts, name.Insecure)
 		}
+
+		// look for wasm/wasi platform
+		for _, p := range bo.Platforms {
```

Comment:
Does this mean that if a user passes `--platform=linux/amd64,wasm/wasi`, that they'll only get the wasi image?

Reply:
Ah good argument, haven't thought about providing multiple architectures with wasi. I'll figure out, if there's a good way when using wasi in multiple platforms.

Reply:
I've filed a request for clarification here: https://github.com/solo-io/wasm/issues/291